src/components/Pages/RequestPage/RequestPage.tsx

```diff
@@ -73,49 +74,37 @@ export const RequestPage = () => {
   }, [requestId])
 
   useEffect(() => {
+    if (isConnecting) return
+
+    if (!account || !provider || !providerType) {
+      console.log("The user isn't logged in, redirecting to the log in page")
```

Comment:
is this log needed?

Reply:
It was part of the old code, but I've removed it.